lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
+
+util.inherits(VOLAPI, RestifyClient);
+
+
+VOLAPI.prototype.close = function close() {
+    this.client.close();
+};
+
+VOLAPI.prototype.createVolume =
+    function createVolume(params, options, callback) {
```

Comment:
> Also discussable is whether all client methods should be strict in validating input options. I'm leaning towards yes.

Do you mean strictly validating the properties of the `options` object passed to `createVolume` and all other exposed methods?